package.json

```diff
@@ -39,10 +39,12 @@
     "i18next-browser-languagedetector": "^6.1.2",
     "i18next-http-backend": "^1.3.1",
     "identity-obj-proxy": "^3.0.0",
+    "install": "^0.13.0",
     "jest": "^27.4.3",
     "jest-resolve": "^27.4.2",
     "jest-watch-typeahead": "^1.0.0",
     "mini-css-extract-plugin": "^2.4.5",
+    "npm": "^8.3.0",
```

Comment:
Please run `npm uninstall npm` to remove this and commit & push the package.json and package-lock.json files without it